jenkins-builds/7.0.UP00-19.03-gpu

```diff
@@ -29,4 +29,4 @@
  papi-wrap-1.0-CrayGNU-19.03.eb       --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP00/gpu/easybuild/tools
  papi-wrap-1.0-CrayCCE-19.03.eb                            --installpath=$APPS/UES/jenkins/7.0.UP00/gpu/easybuild/tools
  papi-wrap-1.0-CrayIntel-19.03.eb                          --installpath=$APPS/UES/jenkins/7.0.UP00/gpu/easybuild/tools
- likwid-4.3.3-perf_event.eb           --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP00/gpu/easybuild/tools
+ likwid-4.3.3-perf_event.eb           --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP00/gpu/easybuild/tools
```

Comment:
Do a checkout from the master for these files. This PR should not touch them.

Reply:
@vkarak I have removed the production lists from this pull request